src/rust/Cargo.toml

```diff
@@ -54,18 +54,17 @@ itertools = "=0.11.0"
 # to keep the number of hosts down when and if it's easy to, but having the
 # number grow gradually is also not the end of the world.
 
-[dependencies.soroban-env-host-p22]
-version = "=22.0.0"
-git = "https://github.com/stellar/rs-soroban-env"
-package = "soroban-env-host"
-rev = "5ed156b7fb5dd3e46635aa0f818d5db33dfd6b6a"
-#path = "../../../rs-soroban-env/soroban-env-host"
+# [dependencies.soroban-env-host-p22]
```

Comment:
With this new setup, would it still be possible to build core with a local patch of env?

Reply:
you'd need to move the submodule to a different version, I think. it wouldn't use a patch.

Reply:
It might be easier if we can figure out the right options to remove the need for the copying of the envs into the same path. That way you could at least just point the build to use an env located elsewhere on your filesystem.

I believe the way to do it is with the `--remap-path-prefix` option. @brson posted some information about it over here on the stellar-cli repo because we're also trying to resolve reprocubile build issues in the cli: https://github.com/stellar/stellar-cli/issues/1445.